### ISSUE TITLE
anaconda install spacy 

### ISSUE THREAD

**Issue by jackie2jet** [explosion/spaCy#2347]
I installed spacy and all the affiliated packages via anaconda,but untill now I still can't import any module or library. could anyone shed some light on the problem?
## Your Environment
* Operating System:Windows 10
* Python Version Used:python 3.6
* spaCy Version Used:spacy 2.0
* Environment Information: 

![import error](https://user-images.githubusercontent.com/3447544/40310863-b8be641a-5d40-11e8-933d-79ca8acd0a08.png)

**Comment by ines** [comment]
This is likely not related to spaCy but to your conda environment. Here's my reply from #2273, which reports a similar problem:

> It's also likely not related to spaCy but rather to the conda environment or your `PYTHONPATH` (which can be quite frustrating sometimes, so I definitely feel your pain). 

> For example, you might be executing the wrong Python by accident or the wrong Anaconda environment (and not the one you previously installed spaCy in). This thread has some more background and solutions: https://stackoverflow.com/questions/39449725/python-modul-not-found-after-anaconda-installation (If you search for "ModuleNotFoundError conda", you'll also find other threads and suggestions!)

You can find various topics and solutions [on StackOverflow](https://stackoverflow.com/search?q=anaconda+no+module+named).

**Comment by jackie2jet** [comment]
so I uninstall conda and install spacy by pip,and still I get errors (I did install vc 14.0 )
and I have to uninstall and still not working and I can't uninstall it either! 
So,is it a registry or path issue,I have no clue now，so could any one offer some solutions about this ?
Thanks in advance!

**Comment by jackie2jet** [comment]
![default](https://user-images.githubusercontent.com/3447544/40417442-582e9c78-5eb2-11e8-883c-7869486fdaf8.png)
here's the issue said I need the vc 14 library but I did install it 
![vc14](https://user-images.githubusercontent.com/3447544/40417591-ada9db54-5eb2-11e8-8752-cafe13b1b505.png)

**Comment by ines** [comment]
Okay, so as the error message says, for some reason it can't find the Visual C++ Build Tools. Is the thing you have installed the same as the [build tools you can download from this page](http://landinghub.visualstudio.com/visual-cpp-build-tools)? 

The Microsoft tools unfortunately all have pretty confusing names (at last I often get confused because they all sounds pretty similar). So It's possible that you don't actually have the right tools installed. This seems to be a common issue with the build tools ([see here](https://stackoverflow.com/questions/50408212/rust-cant-find-microsoft-c-build-tools) for example – there's a lot of [similar stuff](https://www.google.com/search?q=can%27t+find+visual+c%2B%2B+build+tools+site:stackoverflow.com&newwindow=1&client=firefox-b-ab&sa=X&ved=0ahUKEwjbsuiP-5vbAhVFhqQKHYcBCicQrQIINCgEMAA&biw=1440&bih=775) on StackOverflow, too).

**Comment by jackie2jet** [comment]
Yes,It's exactly what I downloaded and nothing seem changed, the error keeps existing.

**Comment by jackie2jet** [comment]
thanks！I reinstalled anaconda and after several retries it finally worked

**Comment by jackie2jet** [comment]
thanks！I reinstalled anaconda and after several retries it finally worked
